internal/handler/handler.go

```diff
@@ -33,6 +33,81 @@ type userMgmtSvc struct {
 	logic logic.UserMgmtSvcLogicIer
 }
 
+func validatePass(pass string) *respModel.Response {
+	if len(pass) < 8 {
+		return &respModel.Response{
+			Status:  http.StatusBadRequest,
+			Message: "Password must be 8 characters long",
+			Data:    nil,
+		}
+
+	}
+	done, err := regexp.MatchString("([a-z])+", pass)
+	if err != nil {
+		log.Error(err)
+		return &respModel.Response{
+			Status:  http.StatusInternalServerError,
+			Message: "Failed to match password",
```

Comment:
The failure/error messages to the user should follow the code format. This is for all error/failure cases.
Please update accordingly.

Reply:
done